core/whistle_apps-1.0.0/src/whapps_speech.erl

```diff
@@ -290,6 +354,37 @@ create_response(_Engine, {'error', _R}) ->
 create_response(_Engine, {'ibrowse_req_id', ReqID}) ->
     lager:debug("speech file streaming as ~p", [ReqID]),
     {'ok', ReqID};
+create_response(<<"voicefabric">> = _Engine, {'ok', "200", Headers, Content}) ->
+    [lager:debug("hdr: ~p", [H]) || H <- Headers],
+    lager:debug("converting media"),
+    RawFile = tmp_file_name(<<"raw">>),
+    WavFile = tmp_file_name(<<"wav">>),
+    _ = file:write_file(RawFile, Content),
+    From = "raw -r 8000 -e signed-integer -b 16",
```

Comment:
@jamesaimonetti 
I tried your patch and still got a warning message and timeout.
Debug message in timeout clause looks like: `sending error response: [[[],WarnMsg],WavData]`
I tried to remove `'stderr_to_stdout'` from `PortOptions` and it removes WarnMsg, but function still returns `{'error', ...}` and function returns after timeout. In this case I recieve WAV data without prepending but with incorrect header

The exact warning message is: `sox WARN wav: Length in output .wav header will be wrong since can't seek to fix it.`
In shell you can get this message running sox using pipes on both sides: http://sox.10957.n7.nabble.com/run-sox-in-a-pipe-with-header-based-formats-tt1469.html#a1472

So if we trying to convert raw data using erlang port:
1. `wh_media_util:normalize_media/4` will return {'error', WavData}.
2. `WavData` will have incorrect length in header.
3. `wh_media_util:normalize_media/4` will return after timeout, not after processing all data.
4.`wh_media_util:normalize_media/4` may return before processing all data.